iOSDeviceManager/Utilities/DeviceUtils.m

```diff
@@ -112,39 +112,133 @@ + (NSComparisonResult)comparePreferredSimulator:(FBSimulator *)sim to:(FBSimulat
     NSString *simDeviceName = [sim name];
     NSString *otherSimDeviceName = [otherSim name];
 
-    if ([simVersion isGreaterThan:otherSimVersion]) {
```

Comment:
Let's implement something less complicated.

Reply:
replaced with the method used in run_loop